File: Selenium/Activities/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("The title is :"+driver.title)
    driver.find_element(By.ID,"username").send_keys("admin")
    driver.find_element(By.NAME,"Password").send_keys("password")
    driver.find_element(By.XPATH,"//button[@type='submit']").click()
    driver.quit()
	    

File: Selenium/Activities/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity2 {

	public static void main(String[] args){
		WebDriver driver=new ChromeDriver();
	    driver.get("https://v1.training-support.net/selenium/login-form");
	    System.out.println("The title is :"+driver.getTitle());
	    driver.findElement(By.id("username")).sendKeys("admin");
	    driver.findElement(By.name("Password")).sendKeys("password");
	    driver.findElement(By.xpath("//button[@type='submit']")).click();
	    driver.quit();
	    
	    
	}
}
